include/sys/zfs_znode.h

```diff
@@ -66,6 +66,12 @@ extern "C" {
  */
 #define	ZFS_PROJID		0x0000800000000000ull
 
+/*
+ * Mask to scrub the lower bits of file level attributes
+ */
+#define	ZFS_KEEP_ADDITIONAL_FLAGS	0xFFFFFFFF00000000ull
+
+
```

Comment:
This should be moved into zpl_file.c since it is only intended for use there.

Reply:
Updated.